### PR TITLE
Release 1.3.0

### PR DESCRIPTION
**Pull request by prabhatkumarthakur**
Added helm chart for DOP 1.3.0

### CODE REVIEW

**Comment by prabhatkumarthakur** [comment]
@avishnu @atulabhi @imazik

File: 1.3.0/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+version: 1.3.0
+name: mayaonprem
+appVersion: 1.3.0
+description: This is an on-premise installation of MayaOnline
```
Comment (atulabhi) [inline 1.3.0/Chart.yaml]: this should be director online

File: 1.3.0/templates/ing-maya-auth-ingress.yaml [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+          path: /echo/
+        - backend:
+            serviceName: maya-ui
+            servicePort: 80
+          path: /ui/
+        - backend:
+            serviceName: chat-server
+            servicePort: 8080
+          path: /chat-server/
+        - backend:
```
Comment (atulabhi) [inline 1.3.0/templates/ing-maya-auth-ingress.yaml]: this rule is not required

Comment (prabhatkumarthakur) [reply 1.3.0/templates/ing-maya-auth-ingress.yaml]: It is already present in 1.2.0 also

**Pull request merged**